### DISCUSSION TITLE
屏蔽特定UP主的动态、视频评论区

### DISCUSSION THREAD

**Discussion by sharkucn** [ywmoyue/biliuwp-lite#707] [feature]
### 问题描述

看到#698提及 推荐页、搜索页、动态页内容过滤，是否可以添加“按UP主屏蔽动态、视频评论区”的功能，在查看该UP的动态、视频时不加载评论区？（希望将某些关注账户限定在资讯更新用途）

### 描述解决方案

如图所示：
视频播放界面
![dis1](https://github.com/user-attachments/assets/53e32887-f143-4b03-b56e-e93e68b38f4b)

动态界面
![dis2](https://github.com/user-attachments/assets/674ab8eb-7abc-4b9f-8402-c96191fcd515)
![dis3](https://github.com/user-attachments/assets/c62955a9-522a-4532-bf59-aa3bb8e807da)

### 附加信息

由于msix安装4.6.33.3407时提示无法安装框架，未能上手现有的内容过滤，如果帖子内容已经实现烦请直接删除💦

### 代码贡献

否

**Comment by ywmoyue** [comment]
不是很能理解这个需求，不想看直接不点开对应的评论区就行呀

**Reply by sharkucn** [reply]
你是对的😂
下午思路光从内容过滤出发了💦

另外感谢安装问题的答疑

**Comment by ywmoyue** [comment]
4.6.33.3406之后关闭了x64架构的nativeToolchain编译选项，可能需要先卸载应用重启系统后再安装

如果需要备份设置，可以使用开发者选项中的导出导入设置功能
